### DISCUSSION TITLE
Custom form field data to send to the server

### DISCUSSION THREAD

**Discussion by tbattur22** [dropzone-ui/dropzone-ui-react#475] [Q&A]
Is there any way to set custom form field data in addition to file when uploading the file to the server in post method? As far as I can see there is no way atm unless I missed it. My question is is there any plan to allow it in the near future? Thanks.

**Comment by JinSSJ3** [comment]
Hey there @tbattur22 !

We didn't considered that feature so far, but it sounds great and we'll add it in a near future.
For that we kindly assk you to specify more about this. 
For instance, in order to be clear 👀 , are you talking about to upload the file with some aditional data like:
```jsx
const additionalData = {
//.. any kind of object
}
``` 
Other questions are: 
- Are you talking about to upload just one file with that additional data 
- Or do you mean uploading the complete list of files each one with that additional data? 
- Or maybe do you need to upload the additional data just once?

Thanks in advance.
